processor/processor.go

```diff
@@ -284,6 +285,8 @@ func (proc *HandleT) Status() interface{} {
 		proc.dedupHandler.PrintHistogram()
 	}
 
+	statusRes["pending-events-count"] = append(statusRes["pending-events-count"], proc.multitenantI.Status())
```

Comment:
Should the processor report in its own status the number of pending events in router & batch router queues?

Reply:
yeah
this is a placeholder for now(put it here because the metric call is made from the processor as well)
I'll move it to the respective queues.